### ISSUE TITLE
better xray integration (nice to have)

### ISSUE THREAD

**Issue by recursix** [ServiceNow/AgentLab#132]
* start xray in parallel when launching study.run (if not already started). Provide a clickable link in the stdout
* make the refresh button work for refreshing the results of the currently selected experiments
* when results are loaded, don't select the row 1 of task and seed tables, but select the first available one
